pydicom/tests/test_dataelem.py

```diff
@@ -9,12 +9,14 @@
 import pytest
 
 from pydicom.charset import default_encoding
+from pydicom.datadict import tag_for_keyword
 from pydicom.dataelem import (
```

Comment:
Was tag_for_keyword to be dropped in this PR, or is it folded into the other one?

Reply:
I remove it in the other one, forgot it here. Will change this.